play/src/front/Components/ReportMenu/ReportMenu.svelte

```diff
@@ -80,7 +80,9 @@
     <section>
         {#if blockActive}
             <BlockSubMenu {userUUID} {userName} />
-            <ReportSubMenu {userUUID} />
+            {#if connectionManager.currentRoom?.key.includes("/@/")}
```

Comment:
We should really never do tests on the URL with regexes (except in the admin or in the "mock admin" in the pusher).
An admin could decide that rooms with /_/ are actually private.

Instead use the "canReport" property to know if we can report:

```suggestion
            {#if connectionManager.currentRoom?.canReport}
```